src/components/ConferenceFilter/ConferenceFilter.js

```diff
@@ -5,10 +5,12 @@ import {TYPES} from '../config';
 
 export default class ConferenceFilter extends Component {
   getUrl = (filters) => {
-    const {type} = this.props;
+    const {type, country} = this.props;
 
     if (filters.country) {
       return `/${filters.type || type}/${filters.country}`;
+    } else if (country) {
```

Comment:
That makes the `All country` link not work. If you have `France` selected the link for `All countries` will be `/xxx/france`
- `filters.country` will be null but `props.country` will not so it will return `/javascript/france`

See: https://github.com/darahak/confs.tech/blob/ad395dce8426abbae08c2917bc4e2118d55233c2/src/components/ConferenceFilter/ConferenceFilter.js#L69

Reply:
It should be fixed now.
I also fixed a problem where the fallback was triggered randomly because of a race condition with the fetch requests.

Reply:
Nice, thanks for your work! I'll review that asap!